src/styles.css

```diff
@@ -8,8 +8,8 @@ body {
   flex-direction: column;
   flex-wrap: wrap;
   border: 2px solid black;
-  margin-left:auto;
-  margin-right:auto;
+  margin-left: auto;
+  margin-right: auto;
 }
```

Comment:
- [ ] Kindly improve your styling. Move the content of the wireframe to be closer to the top of the page so one does not need to scroll before one can see the page content.

  ![image](https://github.com/AbdelazizSoliman/Leaderboard/assets/54848363/ead340e1-4d4b-4481-8ca8-59ed9a1e9f23)